build-support/githooks/prepare-commit-msg

```diff
@@ -24,15 +24,17 @@ NUM_RUST_FILES=$(echo "${CHANGED_FILES})" | grep -c -E \
   -e "^rust-toolchain" \
   -e "^src/python/pants/engine/internals/native.py" \
   -e "^build-support/bin/native" \
-  -e "^3rdparty/python/requirements.txt")
+  -e "^3rdparty/python/requirements.txt" \
+  -e "^build-support/bin/generate_travis_yml.py")
```

Comment:
This fixes invalidation so that we run all of CI when `generate_travis_yml.py` changes.